expan/core/early_stopping.py

```diff
@@ -56,7 +59,9 @@ def group_sequential(x,
             the end of experiment
         alpha: type-I error rate
         cap: upper bound of the adapted z-score
-
+        multi_test_correction: multiple correction flag
+        num_tests: number of tests or reported kpis used for multiple correction (default: 1, no correction is done)
```

Comment:
how about "num_tests (integer): number of tests or reported kpis used for multiple correction. This value is only used if multi_test_correction=True"?

Reply:
yes, if multi_test_correction is True I add worker_args['num_tests'] = len(self.report_kpi_names) if _delta. I made that because I don't want to make num_tests as a parameter of the method, because it's basically is the number of reported kpis.

Reply:
I see. 👍